security/qatlib_rbac.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: intel-qat
+  namespace: intel-qat
+---
+apiVersion: security.openshift.io/v1
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities: 
+- IPC_LOCK
+defaultAddCapabilities: null
+fsGroup:
+   type: MustRunAs
+groups: []
+kind: SecurityContextConstraints
+metadata:
+  annotations:
+    kubernetes.io/description: 'SCC to use IPC_LOCK capability for qatlib pod'
```

Comment:
Can we change to
SCC for qatlib based workload

Reply:
DPDK QAT needs it too

Reply:
We can change it to
SCC for Intel QAT technology based workload

Reply:
done. please check